docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -371,29 +390,28 @@ return the following output:
 --------------------------------------------------
 {
   ...
-  "aggregations": {
-    "prices": {
+    "aggregations": {
+    "latency_buckets": {
       "buckets": [
         {
-          "key": 0.0,
-          "doc_count": 18
+          "key": 0,
+          "doc_count": 2
         },
         {
-          "key": 5.0,
-          "doc_count": 48
+          "key": 5,
+          "doc_count": 2
         },
         {
-          "key": 10.0,
-          "doc_count": 25
+          "key": 10,
+          "doc_count": 2
         },
         {
-          "key": 15.0,
-          "doc_count": 6
+          "key": 15,
+          "doc_count": 1
         }
       ]
     }
-  }
-}
+...
 --------------------------------------------------
 // TESTRESPONSE[skip:test not setup]
```

Comment:
I think we should add a test here. This response is incorrect, and that would have been caught by having a test. Let me know if you need a hand with that.